### PR TITLE
Move sheaves out of experimental.

### PR DESCRIPTION
**Pull request by simonbrandhorst**
- **tuning and fixes for elliptic surfaces**
- **move sheaves out of experimental**
- **add some stub docs for sheaves**

### CODE REVIEW

**Comment by fingolfin** [comment]
We want to mess around with authorship, please don't merge.

**Comment by simonbrandhorst** [comment]
please merge. @afkafkafk13 @HechtiDerLachs

File: src/Rings/mpolyquo-localizations.jl [hunk 1]
```diff
@@ -2046,7 +2046,7 @@ end
   R_simp, iso, iso_inv = simplify(R) # This usually does not cost much
   I_simp = ideal(R_simp, restricted_map(iso).(lifted_numerator.(gens(I))))
   J = pre_image_ideal(I_simp)
-  pre_result = ideal(R_simp, [g for g in R_simp.(gens(radical(J))) if !iszero(g)])
+  pre_result = ideal(R_simp, [R_simp(g) for g in gens(radical(J)) if !iszero(g)])
```
Comment (simonbrandhorst) [inline src/Rings/mpolyquo-localizations.jl:2049]: here is something new

File: src/exports.jl [hunk 2]
```diff
@@ -88,6 +94,8 @@ export Halfspace
 export Hecke
 export HilbertData
 export Hyperplane
+export IdealSheaf
+export IdealSheaf
```
Comment (simonbrandhorst) [inline src/exports.jl]:
```suggestion
```

File: src/exports.jl [hunk 3]
```diff
@@ -347,6 +365,9 @@ export codomain
 export codomain_covering
 export coefficient_field
 export coefficient_ring
+export coefficient_ring
```
Comment (simonbrandhorst) [inline src/exports.jl]:
```suggestion
```

File: src/exports.jl [hunk 4]
```diff
@@ -377,6 +398,8 @@ export complete_bipartite_graph
 export complete_graph
 export complex_projective_plane
 export components
+export components
+export components
```
Comment (simonbrandhorst) [inline src/exports.jl]:
```suggestion
```

File: src/exports.jl [hunk 5]
```diff
@@ -1242,6 +1277,7 @@ export projective_special_unitary_group
 export projective_symplectic_group
 export projective_unitary_group
 export projectivization
+export projectivization
```
Comment (simonbrandhorst) [inline src/exports.jl]:
```suggestion
```

File: src/exports.jl [hunk 6]
```diff
@@ -1310,6 +1346,9 @@ export rem_vertices!
 export renest
 export repres
 export representative
+export representative
```
Comment (simonbrandhorst) [inline src/exports.jl]:
```suggestion
```

File: src/exports.jl [hunk 7]
```diff
@@ -1322,6 +1361,7 @@ export restricted_map
 export restricted_map_type
 export restriction
 export restriction_map
+export restriction_map
```
Comment (simonbrandhorst) [inline src/exports.jl]:
```suggestion
```

File: src/exports.jl [hunk 8]
```diff
@@ -1441,6 +1484,8 @@ export subgroup_classes
 export subquo_type
 export subquotient
 export subscheme
+export subscheme
```
Comment (simonbrandhorst) [inline src/exports.jl]:
```suggestion
```

File: src/exports.jl [hunk 9]
```diff
@@ -1521,6 +1571,7 @@ export vamos_matroid
 export vanishing_ideal
 export vanishing_sets
 export variety
+export variety
```
Comment (simonbrandhorst) [inline src/exports.jl]:
```suggestion
```

File: experimental/Schemes/src/elliptic_surface.jl [hunk 10]
```diff
@@ -1517,13 +1516,20 @@ function extended_ade(ADE::Symbol, n::Int)
   return -G, kernel(G; side = :left)
 end
 
+@doc raw"""
+    basis_representation(X::EllipticSurface, D::WeilDivisor)
+
+Return the vector representing the numerical class of `D` 
+with respect to the basis of the ambient space of `algebraic_lattice(X)`.
+"""
```
Comment (simonbrandhorst) [inline experimental/Schemes/src/elliptic_surface.jl:1524]: and here is another new part

**Review by simonbrandhorst — COMMENTED** [review]
99% of this PR is just moving files. I highlighted the few places where it does more.

**Review by HechtiDerLachs — APPROVED** [review]
If tests pass, this is OK.

**Pull request merged**